language/en.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Should I also remove @otacke 's changes to all lang files and German translation?